src/benchmarks/gc/GC.Infrastructure/GC.Infrastructure.Core/TraceCollection/TraceCollector.cs

```diff
@@ -40,7 +39,14 @@ public sealed class TraceCollector : IDisposable
         {
             { CollectType.gc, "gcCollectOnly" },
             { CollectType.cpu,  "collect_cpu" },
-            { CollectType.threadtime, "collect_threadTime" },
+            { CollectType.threadtime, "collect_threadTime" }
+        };
+
+        internal static readonly Dictionary<CollectType, string> LinuxLocalRunCollectTypeMap = new()
+        {
+            { CollectType.gc, "--clrevents gc" },
```

Comment:
Please check: https://github.com/Maoni0/mem-doc/blob/master/doc/.NETMemoryPerformanceAnalysis.md#how-to-collect-top-level-gc-metrics - seems like the right params are: `--profile gc-collect`.